CDCgov/prime-reportstream
# 11267 - removal of legacy pages

This PR ...

Is a catch all for all the work needed to deprecate and archive the legacy pages/links

Test Steps:
1. Log into RS
2. Make sure all pages and links in the site are functioning correctly.

## Changes
- Removed obsolete pages
- Updated urls to fit with the new redesign

## Checklist

### Testing


## Linked Issues
- Fixes #11267
